script/bootstrap

```diff
@@ -18,6 +15,16 @@ else
     fi
 fi
 
+VENV="${VENV:-syrupy.venv}"
+test -d $VENV || python3 -m venv $VENV || return
+. $VENV/bin/activate
```

Comment:
Do we need to use the venv in CI?

Reply:
Originally added in case I needed to uninstall all requirements. Technically don't need it now, but it does make it easier to work with dependencies. It's an extra line wherever we execute a script, so I don't see any harm in it.

Reply:
Personally prefer not having the extra line, makes it cleaner imo (nit pick though)

Reply:
It's also testing the bootstrap script for local dev in a sense. Don't feel too strongly about it.